src/js/background.js

```diff
@@ -28,16 +26,113 @@ var loading = false;
 var intervalId;
 var iconIndex = 0;
 
-function startRecording(id) {
-  tabId = id;
+let blobList = []
+let imageList = []
 
-  startScreenRecording(id);
-  startConsoleRecording(id);
-  startNetworkRecording(id);
+let logs = {}
+let filtered = {
+  console: [],
+  network: []
 }
 
+// for screent shot
+var screenshot = {
+  content: document.createElement("canvas"),
+  data: '',
+
+  init: function () {
+    this.initEvents();
+  },
+
+  saveScreenshot: function (tabs) {
+    var image = new Image();
+    image.onload = function () {
```

Comment:
Remove image canvas code since it is not needed

Reply:
removed